x/circuit/features/msg_trip.feature

```diff
@@ -0,0 +1,74 @@
+Feature: MsgTripCircuitBreaker
+
+	Circuit breaker can disable message execution:
+	- when the caller trips the circuitbreaker for a message or messages
+	- when the permissions are valid
+
+	Rule: the caller must have SUPER_ADMIN permission to trip the circuit
```

Comment:
This isn't really a rule because it's not always true - other permissions levels can trip too. So the rule might be either "a user with SUPER_ADMIN can trip" or "called needs one of X, Y, Z permission"